Write page on misoprostol intervention for maternal hemorrhage (#1667)
ihmeuw/vivarium_research: docs/source/models/intervention_models/intrapartum/misoprostol_intervention.rst

.. _misoprostol_intervention:

============================================
Misoprostol for treating maternal hemorrhage
============================================

.. contents::
   :local:
   :depth: 1

.. list-table:: Abbreviations
  :widths: 15 15 15
  :header-rows: 1

  * - Abbreviation
    - Definition
    - Note
  * - BEmONC
    - Basic emergency obstetric and neonatal care
    - Operationalized as facilities without C-section capabilities
  * - CEmONC
    - Comprehensive emergency obstetric and neonatal care
    - Operationalized as facilities with capabilities to perform  C-section
  * - PAF
    - Population Attributable Fraction
    - 
  * - RR
    - Relative Risk
    - 
  * - ANC
    - Antenatal Care 
    - 
  * - PPH
    - Postpartum Hemorrhage
    - For consistency with the official GBD Cause list and our :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>`,
      we use the term 'maternal hemorrhage' instead of PPH throughout the course of this document.

Intervention Overview
-----------------------

> Misoprostol is an intramuscular prophylactic prostaglandin that can be taken orally or sublingually during labor to prevent or reduce incidence of maternal hemorrhage. Because 

Reviewer: ```suggestion
Misoprostol is a prophylactic prostaglandin that can be taken orally or sublingually during labor to prevent or reduce incidence of maternal hemorrhage. Because 
```

I think it's a bit confusing to have "intramuscular" here given that the intervention as we are modeling it is administered orally (or vaginally, I believe!)

receive an advanced distribution of misoprostol for self-administration during a home birth. Because misoprostol is not as effective as conventional 
uterotonics (i.e., oxytocin) in prevention of maternal hemorrhage, we will not model the distribution of misoprostol in BEmONC or CEmONC facilities where injectible 
uterotonics are more widely available ([Tunçalp-2012-Cochrane-Review]_). 

This section describes how a misoprostol intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
See the :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>` for relevant details.

.. list-table:: Affected Outcomes
  :widths: 15 15 15 15
  :header-rows: 1

  * - Outcome
    - Effect
    - Modeled?
    - Note (ex: is this relationship direct or mediated?)
  * - Maternal Hemorrhage Incidence Probability :math:`\text{IR}_i^\text{maternal hemorrhage}`
    - Adjust multiplicatively using RR
    - Yes 
    - 
      - :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>` 
      - For convenience, we will model this like a dichotomous risk factor; more details below

> Baseline Coverage Data

Reviewer: I just forwarded you an email with attachments of all the details in the PPH/home birth model we made years ago. It looks like in that model, we assumed 0% coverage given that this paper said there had not been much scale-up of community distribution when they wrote the paper in 2018 (https://www.tandfonline.com/doi/full/10.1080/17441692.2017.1303743). I'm not sure if there has been more information since then, but maybe it is useful to restrict your searching to studies published after 2018 :)

Reviewer: Also, something else you could try checking is whether misoprostol is included in each country's "safe birth kit" which are ideally distributed at ANC. There may be country-specific policies/guidelines on this that you could check. My understanding is that many countries do not have misoprostol for community distribution programs given the concern that it could be used for abortion. I think we could reasonably assume zero coverage if it's not in this sort of national guideline. 

Here is a paper that looks at clean birth kits in Ethiopia -- maybe the reference a national guideline or something? https://pmc-ncbi-nlm-nih-gov.offcampus.lib.washington.edu/articles/PMC11341469/

Reviewer: https://onlinelibrary-wiley-com.offcampus.lib.washington.edu/doi/10.1111/jmwh.12154

due to not receiving misoprostol.  We will use the decision tree below to estimate the probability of maternal hemorrhage incidence with and without the use of misoprostol, ensuring consistency
with the baseline delivery facility rates and baseline misoprostol coverage.

In Vivarium, this risk effect will modify the maternal hemorrhage incidence pipeline, resulting in 

.. math::

   \text{IR}_i^\text{maternal hemorrhage} = \text{IR}^\text{maternal hemorrhage}_ \cdot (1 - \text{PAF}_\text{no misoprostol}) \cdot \text{RR}_i^\text{no misoprostol}

where :math:`\text{RR}_i^\text{no misoprostol}` is simulant *i*'s individual relative risk for "no misoprostol", meaning :math:`\text{RR}_i^\text{no misoprostol} = \text{RR}_\text{no misoprostol}` 
if simulant *i* does not receive misoprostol, and :math:`\text{RR}_i^\text{no misoprostol} = 1` if simulant *i* receives misoprostol. 

The relative risk value we will use is pulled from [Tunçalp-2012-Cochrane-Review]_, the most recent Cochrane Review of the effect of 

> sublingually received misoprostol during labor on the prevention of maternal hemorrhage.

Reviewer: I happen to be familiar with the literature from the PPH model we made in 2019/2020 -- here is a more recent network meta-analysis we used for that project that has a slightly more effective estimate for misoprostol. I haven't checked if this is relevant to home/oral settings, but if it is it is probably worth updating to this data source. 

https://pubmed.ncbi.nlm.nih.gov/29693726/

Author: Oh thanks for sharing! I'm not sure we want to use the value in the paper you shared though because they report an RR for a 'misoprostol plus oxytocin' combination in hospital settings, rather than for oral misoprostol!

Author: Ahh I see!! Thanks, I missed that!

  with controls (56 studies, 2776/25,647 (10.8%) versus 614/26,800 (2.3%); average RR 3.97, 95% CI 3.13 to 5.04; Tau² = 0.47, I² = 80%). The effect 
  was greater for trials using misoprostol 600 µg or more (27 studies; 2197/17,864 (12.3%) versus 422/18,161 (2.3%); average RR 4.64; 95% CI 3.33 to 
  6.46; Tau² = 0.51, I² = 86%) than for those using misoprostol 400 µg or less (31 studies; 525/6751 (7.8%) versus 185/7668 (2.4%); average RR 3.07; 
  95% CI 2.25 to 4.18; Tau² = 0.29, I² = 58%)".)
- We assume that baseline coverage for misoprostol in home births is 5% (this is not data-backed).
- We do not model use of misoprostol for prevention of maternal hemorrhage in BEmONC and CEmONC facilities based on the [WHO-2020]_ recommendation and with
  the assumption that BEmONC and CEmONC facilities in our locations of interest have injectible uterotonics such as oxytocin widely available for the 
  prevention of maternal hemorrhage. (Note: in 2020, our team did a literature review of uterotonic coverage in facility settings in LMICs, the DRC, India, 

>   Kenya, and Nigeria, and found 0-1% of facilities had misoprostol available for PPH prevention and 43-94% of facilities had oxytocin available).

Reviewer: It would be great to have some sort of reference to this so we can look back at it, even it's just the J drive link!

with the baseline delivery facility rates and baseline misoprostol coverage.

In Vivarium, this risk effect will modify the maternal hemorrhage incidence pipeline, resulting in 

.. math::

   \text{IR}_i^\text{maternal hemorrhage} = \text{IR}^\text{maternal hemorrhage}_ \cdot (1 - \text{PAF}_\text{no misoprostol}) \cdot \text{RR}_i^\text{no misoprostol}

where :math:`\text{RR}_i^\text{no misoprostol}` is simulant *i*'s individual relative risk for "no misoprostol", meaning :math:`\text{RR}_i^\text{no misoprostol} = \text{RR}_\text{no misoprostol}` 
if simulant *i* does not receive misoprostol, and :math:`\text{RR}_i^\text{no misoprostol} = 1` if simulant *i* receives misoprostol. 

The relative risk value we will use is pulled from [Tunçalp-2012-Cochrane-Review]_, the most recent Cochrane Review of the effect of 
sublingually received misoprostol during labor on the prevention of maternal hemorrhage.

.. list-table:: Risk Effect Parameters for No Misoprostol
  :widths: 15 15 15 15
  :header-rows: 1

  * - Parameter
    - Mean
    - Distribution
    - Notes
  * - Relative Risk
    - 1.52
    - :math:`\text{Normal}(1.52,0.08^2)`

>     - Based on relative risk of 0.66 (95% CI 0.10-0.94) on maternal hemorrhage incidence for pregnant people receiving misoprostol

Reviewer: Similar to azithromycin, let's update this uncertainty interval too https://github.com/ihmeuw/vivarium_research/pull/1673
